composer.json

```diff
@@ -25,10 +25,10 @@
         "sonata-project/doctrine-extensions": "^1.0",
         "sonata-project/easy-extends-bundle": "^2.1",
         "sonata-project/google-authenticator": "^1.0",
-        "symfony/console": "^2.3",
-        "symfony/form": "^2.3",
-        "symfony/http-foundation": "^2.3",
-        "symfony/security": "^2.3"
+        "symfony/console": "^2.3.5 || ^3.0",
+        "symfony/form": "^2.3.5 || ^3.0",
+        "symfony/http-foundation": "^2.3.5 || ^3.0",
+        "symfony/security": "^2.3.5 || ^3.0"
```

Comment:
[Here](https://travis-ci.org/sonata-project/SonataUserBundle/jobs/175058819#L727) are some problems that need to be fixed before we can merge this.

```
Accessing type "sonata_security_roles" by its string name is deprecated since version 2.8 and will be removed in 3.0. Use the fully-qualified type class name "Sonata\UserBundle\Form\Type\SecurityRolesType" instead: 6x

    2x in SecurityRolesTypeTest::testSubmitValidData from Sonata\UserBundle\Tests\Form\Type

    2x in SecurityRolesTypeTest::testSubmitInvalidData from Sonata\UserBundle\Tests\Form\Type

    2x in SecurityRolesTypeTest::testSubmitWithHiddenRoleData from Sonata\UserBundle\Tests\Form\Type

Accessing type "choice" by its string name is deprecated since version 2.8 and will be removed in 3.0. Use the fully-qualified type class name "Symfony\Component\Form\Extension\Core\Type\ChoiceType" instead: 3x

    1x in SecurityRolesTypeTest::testSubmitValidData from Sonata\UserBundle\Tests\Form\Type

    1x in SecurityRolesTypeTest::testSubmitInvalidData from Sonata\UserBundle\Tests\Form\Type

    1x in SecurityRolesTypeTest::testSubmitWithHiddenRoleData from Sonata\UserBundle\Tests\Form\Type

The value "false" for the "choices_as_values" option of the "choice" form type (Symfony\Component\Form\Extension\Core\Type\ChoiceType) is deprecated since version 2.8 and will not be supported anymore in 3.0. Set this option to "true" and flip the contents of the "choices" option instead: 3x

    1x in SecurityRolesTypeTest::testSubmitValidData from Sonata\UserBundle\Tests\Form\Type

    1x in SecurityRolesTypeTest::testSubmitInvalidData from Sonata\UserBundle\Tests\Form\Type

    1x in SecurityRolesTypeTest::testSubmitWithHiddenRoleData from Sonata\UserBundle\Tests\Form\Type

The Symfony\Component\Form\ChoiceList\ArrayKeyChoiceList class is deprecated since version 2.8 and will be removed in 3.0. Use Symfony\Component\Form\ChoiceList\ArrayChoiceList instead: 1x

    1x in SecurityRolesTypeTest::testSubmitValidData from Sonata\UserBundle\Tests\Form\Type
```